source/usage-examples/insertMany.txt

```diff
@@ -0,0 +1,31 @@
+=====================
+Create Many Documents
+=====================
+
+.. default-domain:: mongodb
+
+You can create multiple documents using the `insertMany()
+<https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#insertMany>`_
+method. The ``insertMany()`` method inserts multiple documents in the specified collection 
+by taking in an array of documents. 
```

Comment:
Minor: I find this phrasing a bit odd. Maybe something like:

>The ``insertMany()`` method *takes an array of documents* to insert into the specified collection.

Reply:
That does sound better - fixed verbatim